src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.Android.cs

```diff
@@ -54,7 +54,8 @@ public new static class TraceProvider
 
 		static TextBlock()
 		{
-			InitializeStaticLayoutInterop();
+			if ((int)Android.OS.Build.VERSION.SdkInt < 28)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4902768)